src/runtime/manifest.ts

```diff
@@ -785,12 +793,15 @@ ${e.message}
       items.byParticle.set(particle, item);
 
       for (const slotConnectionItem of item.slotConnections) {
+        if (slotConnectionItem.direction === 'provide') {
+          throw new ManifestError(item.location, `invalid slot connection`);
+        }
         let slotConn = particle.consumedSlotConnections[slotConnectionItem.param];
         if (!slotConn) {
           slotConn = particle.addSlotConnection(slotConnectionItem.param);
         }
         slotConn.tags = slotConnectionItem.tags || [];
-        slotConnectionItem.providedSlots.forEach(ps => {
+        slotConnectionItem.dependentSlotConnections.forEach(ps => {
```

Comment:
Do we need to check that the dependentSlotConnections _are_ 'provide' connections?

Reply:
Done. Also made sure that provide slots can not have dependent slot connections. This is to make sure we still have the same restrictions on slot connections as we did before, despite the parser handling these connections differently. 

(brief summary of the restrictions on slot connections)
- There can be 0 or more consume slots
- Each consume slot can have 0 or more dependent provide slots
- Provide slots can not have any dependencies